content/en/docs/concepts/services-networking/endpoint-slices.md

```diff
@@ -135,29 +134,41 @@ For pods, this is any pod that has a deletion timestamp set.
 
 ### Topology information {#topology}
 
-{{< feature-state for_k8s_version="v1.20" state="deprecated" >}}
-
-{{< note >}}
-The topology field in EndpointSlices has been deprecated and will be removed in
-a future release. A new `nodeName` field will be used instead of setting
-`kubernetes.io/hostname` in topology. It was determined that other topology
-fields covering zone and region would be better represented as EndpointSlice
-labels that would apply to all endpoints within the EndpointSlice.
-{{< /note >}}
-
 Each endpoint within an EndpointSlice can contain relevant topology information.
 This is used to indicate where an endpoint is, containing information about the
-corresponding Node, zone, and region. When the values are available, the
-control plane sets the following Topology labels for EndpointSlices:
+corresponding Node, zone. When the values are available, the
+control plane sets the following per endpoint fields for EndpointSlices:
 
-* `kubernetes.io/hostname` - The name of the Node this endpoint is on.
-* `topology.kubernetes.io/zone` - The zone this endpoint is in.
-* `topology.kubernetes.io/region` - The region this endpoint is in.
+* `nodeName` - The name of the Node this endpoint is on.
+* `zone` - The zone this endpoint is in.
 
-The values of these labels are derived from resources associated with each
-endpoint in a slice. The hostname label represents the value of the NodeName
-field on the corresponding Pod. The zone and region labels represent the value
-of the labels with the same names on the corresponding Node.
+#### Deprecated per endpoint `topology` field
+In the GA API, the field `deprecatedTopology` was added for backwards
+compatibility with the `topology` field that exists on the v1beta1 API. The
+`topology` will be converted to `depcreatedTopology` and will be available as
+a read only field for `v1` clients. Any attempts by a `v1` client to write to
+`deprecatedTopology` will cause the entire field to dropped. The
```

Comment:
Reword:
Any attempts by a `v1` client to write to `deprecatedTopology` causes the entire field to be ignored?